src/screens/profile/container/profileContainer.js

```diff
@@ -75,7 +76,7 @@ class ProfileContainer extends Component {
   };
 
   componentWillReceiveProps(nextProps) {
-    const { navigation, currentAccount, activeBottomTab, isLoggedIn } = this.props;
+    const { activeBottomTab, currentAccount, isLoggedIn, navigation } = this.props;
     const currentUsername =
       currentAccount.name !== nextProps.currentAccount.name && nextProps.currentAccount.name;
```

Comment:
use get from lodash

Reply:
it has different condition, `get` not suitable for use in here

Reply:
get(nextProps, 'currentAccount.name') @mistikk